pages/blog/graphql-deep-dive-1.mdx

```diff
@@ -0,0 +1,354 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'GraphQL - Diving Deep',
+  author: 'tvvignesh',
+  tags: ['graphql', 'deepdive', 'faq'],
+  date: '2020-11-28',
+  description:
+    'Answering all the questions you may have about GraphQL and its ecosystem',
+  image: '/blog-assets/graphql-deep-dive-1/cover.png',
+  thumbnail: '/blog-assets/graphql-deep-dive-1/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+_This blog is a part of a series on GraphQL where we will dive deep into GraphQL and its ecosystem one piece at a time_
```

Comment:
Would be great to add link to all the other parts of the series in all the articles

Reply:
Sure. Will do it. Was planning to do it after writing all the blogs - will add links to the blogs added so far.